docs/examples/sujoy_edge/sujoy_edge_demo.jl

```diff
@@ -0,0 +1,80 @@
+# ---
+# cover: assets/sedge.png
+# title: Sujoy Edge Demo
```

Comment:
Would you mind to give a more meaningful title here? For example, it could be "Image edge detection using Sujoy kernel".

Also, it could be nice to provide a `author` and `date` meta here. For example:

```text
# author: Sujoy Kumar Goswami; Partha Ghosh
# date: 2020-09-21
```

Reply:
Fix this